### DISCUSSION TITLE
LCD not displaying any characters

### DISCUSSION THREAD

**Discussion by Stulek** [GeekFunkLabs/fluidpatcher#84] [Q&A] [answered]
![2023-10-09 17 13 07](https://github.com/GeekFunkLabs/fluidpatcher/assets/125310322/a4475a3a-2d69-4935-b2fe-42a1ebc45f01)

Hi everyone

I build SquishBox two resistor PCB edition but not work.
I follow instruction step by step
I charge other OS and other RPI but not starting
I have AKAI mini MK3 (headles version works well)
web function available
pls help

![rpi](https://github.com/GeekFunkLabs/fluidpatcher/assets/125310322/74bda23c-a6ef-4672-a5e4-28643757e292)

Thanks

**Comment by albedozero** [comment]
Sorry you're having trouble. Having the web interface available means the software is working. Seeing a single row of solid boxes that never changes means that the software is not talking to the LCD correctly. Here are a couple things to try first:
- Are you sure that when running the install script `curl -L git.io/squishbox | bash` you chose the option `v4 - Purple PCB, has 2 resistors and LED`? To check, you can paste this command to your Raspberry Pi:
```bash
grep ^HW_VERSION squishbox.py
```
It should return `HW_VERSION = 'v4'`
- If you plug in headphones and a MIDI keyboard and play notes, do you hear any sound?

**Comment by Stulek** [comment]
I chooese V4 and not work :-((

![rpi](https://github.com/GeekFunkLabs/fluidpatcher/assets/125310322/d0742d3d-f652-4e4d-a821-f1980b3926a4)

**Comment by albedozero** [comment, marked as answer]
Thanks for providing the details of the error - that's helpful. The choices provided during installation are:
```
v6 - Green PCB with SMT components
v4 - Purple PCB, has 2 resistors and LED
v3 - Purple PCB, has 1 resistor
v2 - Hackaday/perfboard build
```
Your response should be `v4`, not just `4`. I'll try to think of a way to make that more clear. You should be able to fix things with this one-line command:
```
sed -i "/^HW_VERSION/s|.*|HW_VERSION='v4'|" squishbox.py
```

**Reply by Stulek** [reply]
Thanks i setting correct parameter and work well
good support :-)
